WordPress/Docs/WhiteSpace/OperatorSpacingStandard.xml

```diff
@@ -0,0 +1,42 @@
+<documentation title="Operator Spacing">
+    <standard>
+    <![CDATA[
+    Always put a space after commas, and on both sides of logical, comparison, string and assignment operators.
```

Comment:
This phrase still raises some questions for me when I read it:
* Spacing around comma's is not checked by this sniff.
* What are "string operators" ?

For your information, the tokens being checked by this sniff are:
* Comparison operators, i.e. `==`, `!==`, `<=`, `<=>`, `??` etc
* Assignment operators, i.e. `=`, `.=`, `+=` etc
* Operators for arithmetic and bitwise operations, i.e. `+`, `*`, `<<`, `|` etc
* Ternary operators, i.e. `?` and `:` (when used in a ternary).
* Boolean/Logical operators, i.e. `&&`, `and`, `!`, `xor` etc
* The `instanceof` operator.

Does that help to rephrase this sentence ?

Reply:
Thanks for the feedback. I surely don't know any more why I added the comma part.

I shortened and "simplified" the description to `Always put one space on both sides of operators.` - does this fit better?

Reply:
I kind of liked that you were explicit about which operators this sniff deals with.

> I shortened and "simplified" the description to Always put one space on both sides of operators. - does this fit better?

That would be inaccurate as this doesn't apply - and is actually an error for some other operators:
* Increment/decrement - `++$i` - should not have a space between the operator and the variable it applies to.
* Same thing for the spread operator `...$args` (see #1762 )
* AFAIK we currently don't have any rules for spacing around the silence operator `@`.
* Nor do we have rules for the spacing around \` backtick operators, no space on the "inside" is customary, but WPCS bans the use of these anyway.
* As for assignment operators, those can (and often should) be aligned, so while the one space rule applies to the _right_ of the operator, more space is allowed on the _left_.

Does that help ?